tascrum/tests/test_profile.py

```diff
@@ -95,35 +95,39 @@ def test_Meber_fileds(self):
         self.assertEqual(member1.bio, 'Another test bio')
         self.assertEqual(str(member1.birthdate), '1990-05-15')
 
-    # def test_post_profile(self):
-    #     response = self.client.post(reverse("profile-list" , {
-    #         first_name='saba', last_name='razi',email='razi.saba@gmail.com',\
-    #                                       username= "sabarzii", password='thisissaba'
-    #     }))
+    def test_post_profile(self):
+        response = self.client.post(reverse("profile-list"), {'first_name': 'saba', 'last_name': 'razi', 'email': 'razi.saba@gmail.com', 'username': 'sabarzii', 'password': 'thisissaba'})
+
```

Comment:
it would be better to check each part of profile in different tests.

Reply:
ok got it